src/reference/sectionB_elements.inc

```diff
@@ -398,8 +398,8 @@ A :code:`math` element information item (referred to in this specification as a
 +----------------------------------+----------------------------------------------------------------+
 | Qualifier Elements               | :code:`bvar`, :code:`logbase`, :code:`degree` \*               |
 +----------------------------------+----------------------------------------------------------------+
-| Trigonometric Operators          | | :code:`sin`, :code:`cos`, :code:`tan`, :code:`sec`,          |
-|                                  |   :code:`csc`, :code:`cot`,                                    |
+|                                  | | :code:`sin`, :code:`cos`, :code:`tan`, :code:`sec`,          |
+| Trigonometric Operators          |   :code:`csc`, :code:`cot`,                                    |
```

Comment:
Why has it been shifted? Looks odd.

Reply:
Whoops was trying something out and forgot to put it back.